cmd/nanobus/main.go

```diff
@@ -106,28 +105,21 @@ func (c *runCmd) Run() error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	location := c.BusFile
-	if oci.IsImageReference(c.BusFile) {
-		fmt.Printf("Pulling %s...\n", c.BusFile)
-		var err error
-		if location, err = oci.Pull(location, "."); err != nil {
-			fmt.Printf("Error pulling image: %s\n", err)
-			return err
-		}
-
-		if location == "" {
-			// Fallback to default application config filename.
-			location = "bus.yaml"
-		}
+	var reference string
```

Comment:
With `default:"bus.yaml"` in the struct tag above, these lines (108-113) are not necessary...